tests/test_livysession.py

```diff
@@ -39,11 +36,18 @@ class TestLivySession(object):
     log_json = json.loads('{"id":6,"from":0,"total":212,"log":["hi","hi"]}')
 
     def __init__(self):
+        self.http_client = None
```

Comment:
Why make both http_client and spark_events class instances? I really dislike coupling unit tests like this unless there's an extremely good reason.